Marlin/Marlin_main.cpp

```diff
@@ -5004,22 +5085,22 @@ inline void gcode_M42() {
 
     bool stow_probe_after_each = code_seen('E');
 
-    float X_probe_location = code_seen('X') ? code_value_axis_units(X_AXIS) : X_current + X_PROBE_OFFSET_FROM_EXTRUDER;
+    float X_probe_location = code_seen('X') ? code_value_axis_units(X_AXIS) : X_current;
```

Comment:
The coordinates given here are where to move the **_probe_**, not where to move the **_nozzle_**.